content/sql/concepts/math-functions/terms/atan/atan.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This is not a properly formatted metadata section. You can learn how to put together a metadata section in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).

Reply:
ok sir I will do that thank you for reviewing